src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -935,7 +932,7 @@ namespace OpenMS
       pep_hit_.setScore(xx_lda_prescore);
       if (swath_present)
       {
-        pep_hit_.setScore(mrmfeature->getScore("xx_swath_prelim_score"));
+        // pep_hit_.setScore(mrmfeature->getMetaValue("xx_swath_prelim_score"));
```

Comment:
what about this one?

Reply:
good point, is fixed now (this caused issues when the meta value was not present)